js/gem-notice.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
@jonathanbardo @EvanHerman why not just delete the `wpem_gem_notice_option` on `shutdown` if we know it exists? Then the message would fire only once the first time you go to the Admin, and go away with a click of the X or reload of any page.